openlibrary/templates/lib/nav_head.html

```diff
@@ -111,7 +111,7 @@
     <div class="account-component">
       <div class="dropdown-avatar">
         <button class="avatar" id="userToggle" aria-label="My account" style="background-image: url('https://archive.org/services/img/$(get_internet_archive_id(ctx.user.key))');"></button>
-        <img class="down-arrow" aria-hidden="true" alt="$_('My Account')" role="presentation" src="/static/images/down-arrow.png"/>
+        <img class="down-arrow" aria-hidden="true" alt="$_('My Account')" src="/static/images/down-arrow.png"/>
```

Comment:
This can also be be removed, since the preceding element is `aria-label`ed.

Reply:
Should be fine now :+1: